builtin/providers/aws/resource_aws_iam_group_test.go

```diff
@@ -105,3 +112,9 @@ resource "aws_iam_group" "group" {
 	path = "/"
 }
 `
+const testAccAWSGroupConfig2 = `
+resource "aws_iam_group" "group" {
+	name = "test-group2"
+	path = "/funnypath/"
```

Comment:
won't this fail the test? Maybe we could supply `testAccCheckAWSGroupAttributes` with the expected path, and compare them